media-video/shotcut/shotcut-20.06.28.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils xdg
+
+DESCRIPTION="A free, open source, cross-platform video editor"
+HOMEPAGE="https://www.shotcut.org/ https://github.com/mltframework/shotcut/"
+SRC_URI="https://github.com/mltframework/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+"
+COMMON_DEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5[widgets]
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsql:5
+	dev-qt/qtwebkit:5
+	dev-qt/qtwebsockets:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	>=media-libs/mlt-6.18.0[ffmpeg,frei0r,jack,qt5,sdl,xml]
+	media-libs/webvfx
+	media-video/ffmpeg
+"
+DEPEND="${COMMON_DEPEND}
+	dev-qt/qtconcurrent:5
+	dev-qt/qtx11extras:5
+"
+RDEPEND="${COMMON_DEPEND}
+	dev-qt/qtgraphicaleffects:5
+	dev-qt/qtquickcontrols:5
+	virtual/jack
+"
+
+src_prepare() {
+	default
+
+	sed -i -e '/QT.*private/d' \
+		src/src.pro || die
+	REPLACE_PATTERN="Shotcut is a free, open source, cross-platform video editor."
+	sed -i \
+		-e "s/^Comment=Video Editor$/Comment=$REPLACE_PATTERN"/ \
+		-e '/^Comment\[fr\]=/d' \
+		-e '/^Comment\[ru\]=/d' \
+		packaging/linux/org.shotcut.Shotcut.desktop || die
```

Comment:
This is too ugly and probably error-prone, let's make it a patch. Ideally you'd report this to upstream and get it fixed/updated there. They use Github so you can easily make a pull request too, to get it fixed for their next release.

Reply:
Sent a pull request there. Should I revert the changes into the .desktop file for this pull?

Reply:
Please include the PR patch in this ebuild, name it like `shotcut-20.06.28-desktop-validate.patch`. It can be dropped from the next version when upstream fixes it. 

Ideally we'd wait for upstream comments on that PR, or if they merge it so we can get the full commit as a patch, but it looks simple enough already.